gradle/libs.versions.toml

```diff
@@ -154,12 +156,14 @@ hadoop3-common = { group = "org.apache.hadoop", name = "hadoop-common", version.
 hadoop3-client = { group = "org.apache.hadoop", name = "hadoop-client", version.ref = "hadoop3"}
 hadoop3-minicluster = { group = "org.apache.hadoop", name = "hadoop-minicluster", version.ref = "hadoop-minikdc"}
 hadoop3-gcs = { group = "com.google.cloud.bigdataoss", name = "gcs-connector", version.ref = "hadoop3-gcs"}
+hadoop3-oss = { group = "org.apache.hadoop", name = "hadoop-aliyun", version.ref = "hadoop3-aliyun"}
 htrace-core4 = { group = "org.apache.htrace", name = "htrace-core4", version.ref = "htrace-core4" }
 airlift-json = { group = "io.airlift", name = "json", version.ref = "airlift-json"}
 airlift-resolver = { group = "io.airlift.resolver", name = "resolver", version.ref = "airlift-resolver"}
 httpclient5 = { group = "org.apache.httpcomponents.client5", name = "httpclient5", version.ref = "httpclient5" }
 mockserver-netty = { group = "org.mock-server", name = "mockserver-netty", version.ref = "mockserver" }
 mockserver-client-java = { group = "org.mock-server", name = "mockserver-client-java", version.ref = "mockserver" }
+commons-lang = { group = "commons-lang", name = "commons-lang", version.ref = "commons-lang" }
```

Comment:
You need to update the LICENSE.bin for your new added dependency.

Reply:
I have confirmed it and it seems there already exists the content in the LICENSE.bin


https://github.com/apache/gravitino/blob/93cdbc259b1b4bcd4bda2d1b34d011f19368d2dd/LICENSE.bin#L253